.github/workflows/main.yml

```diff
@@ -47,12 +51,15 @@ jobs:
       run: .github/bin/dist-github-release.sh
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    - if: github.event_name != 'pull_request'
+    - if: >-
+        github.event_name != 'pull_request' &&
+        github.repository == 'planetarium/libplanet'
```

Comment:
I believe this change is unnecessary, because it's enough that the `build` job in itself is under the same condition.

Reply:
I found that the [Run .github/bin/dist-nuget.sh](https://github.com/gurrpi/libplanet/runs/511138491?check_suite_focus=true) step failed on the forked repository when triggered by `on: push`.

```
Run .github/bin/dist-nuget.sh
This script requires NUGET_API_KEY envrionment variable
##[error]Process completed with exit code 1.
```

Reply:
That's because [it should be configured in the repository settings.][1]

[1]: https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#creating-encrypted-secrets